### PR TITLE
contact as well as contribution added

### PR DESCRIPTION
**Pull request by Varsani2520**
i added this:

contact page
contribution page
main file enhance /update
do component based folder structure
add react router dom 
and code style changed 

# Issue Title:  #497 
*closes* #497

## Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

 
## How Has This Been Tested? ⚙️

click on contact then redirect contact route as well as team route
image
image

### CODE REVIEW

**Comment by Varsani2520** [comment]
@singodiyashubham87 ,review it and merged it and any query then inform me

**Comment by Varsani2520** [comment]
@singodiyashubham87 ,i request you to please merged it

File: package-lock.json
Comment (singodiyashubham87) [inline package-lock.json:1]: Why have you changed this file?

**Review by singodiyashubham87 — APPROVED** [review]
Approved!

**Pull request merged**